src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body class="d-fle">
+	<main class="board">
+		<h1>Leaderboard</h1>
+		<section class="main d-flex">
+			<div class="scores">
+				<div class="d-flex score-intro">
+					<h2>Recent scores</h2>
+					<div><button type="reset">Refresh</button></div>
+				</div>
+				<ul class="new-scores">
+				</ul>
+			</div>
+			<section class="add-scores d-flex">
+				<h2>Add your score</h2>
+				<div><input type="text" class="name-input" placeholder="Your name"></div>
+				<div><input type="text" class="score-input" placeholder="Your score"></div>
+				<div><input class="submit" type="button" value="Submit"></div>
+			</section>
+		</section>
+	</main>
+</body>
+</html>
```

Comment:
- [ ] Kindly bring your form for adding scores and the recent scores close to match the wireframe as they are so widely apart. See what I mean below:

Your Implementation | Wireframe
:---:|:---:
![](https://i.imgur.com/fxHTgJM.png) | ![](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png?raw=true)